sdk/tests/conformance/more/functions/bufferSubDataBadArgs.html

```diff
@@ -67,8 +67,14 @@
         function(){gl.bufferSubData(gl.ELEMENT_ARRAY_BUFFER, 19, new Uint16Array([1,2,3]));});
     assertFail("data too large",
         function(){gl.bufferSubData(gl.ELEMENT_ARRAY_BUFFER, 0, new Uint16Array(data.concat([1])));});
-    assertOk("offset + data too large",
+    assertOk("offset + data.length = end",
         function(){gl.bufferSubData(gl.ELEMENT_ARRAY_BUFFER, 18, new Uint16Array([1,2,3]));});
+    assertOk("offset Infinity",
+        function(){gl.bufferSubData(gl.ELEMENT_ARRAY_BUFFER, Infinity, new Uint16Array([1,2,3]));});
+    assertOk("offset -Infinity",
+        function(){gl.bufferSubData(gl.ELEMENT_ARRAY_BUFFER, -Infinity, new Uint16Array([1,2,3]));});
+    assertOk("offset NaN",
+        function(){gl.bufferSubData(gl.ELEMENT_ARRAY_BUFFER, NaN, new Uint16Array([1,2,3]));});
```

Comment:
Apologies, I was getting the parameter and original size confused.  @kenrussell is correct, BUFFER_SIZE should return the original buffer size. However, the original buffer has 12 elements and is being filled with a Uint16Array, which has 2 bytes per element.  Since BUFFER_SIZE is in bytes, the total should be 24 bytes, 12. No?